Hardware/HardwareTests/AirPrintReceipt/ReceiptRendererTest.swift

```diff
@@ -40,7 +40,7 @@ private extension ReceiptRendererTest {
                 amount: 1,
                 formattedAmount: "1",
                 currency: "USD",
-                date: .init(),
+                date: .init(timeIntervalSince1970: 1636970486),
```

Comment:
Good catch @jaclync 👍 

Shameless plug: https://mokacoding.com/blog/better-swift-date-compare-tests/ – Using and hardcoded `Date()` can backfire 😄

Reply:
Thank you @jaclync for fixing this! "The first pancake is lumpy" =)